itests/org.openhab.core.io.bin2json.tests/src/test/java/org/openhab/core/io/bin2json/tests/Bin2JsonTest.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.bin2json.tests;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.Test;
+import org.openhab.io.bin2json.Bin2Json;
+import org.openhab.io.bin2json.ConversionException;
+
+import com.google.gson.JsonObject;
+
+/**
+ * Integration tests for {@link Bin2Json}.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class Bin2JsonTest {
```

Comment:
This looks like a pure JUnit test and not a test that requires an OSGi framework.
You don't need to use an integration test project at all.

It should be enough to use that class into a package that resides in the `src/test/java` folder of your bundle.
The JUnit test should then be run on compile time.